### ISSUE TITLE
Is the loss function done?

### ISSUE THREAD

**Issue by ghost** [godelscat/YOLOv3#1]
Hello shim, first at all thank you for your amazing job, and of course for sharing it, this is the first implementation of pytorch-yolov3 I could find with a nice designed architecture.

I tried it, and it works like a charm, but I have the following dumb questions if you have a little time.

- Is the loss function done? it says at one comment that you only compute yolo loss for one scale.
- Is the loss function the same as the YoloV3 paper? since I tried it with a real dataset and I'm having a little trouble training the model, so that idea got into my mind.

Thank you so much in advance for any comment, and thanks again for sharing your outstanding work.

**Comment by godelscat** [comment]
Hi, thanks for your interest. 

> * Is the loss function done? it says at one comment that you only compute yolo loss for one scale.

Yes, the loss function is done, and it's calculated by [yolo_loss](https://github.com/godelscat/YOLOv3/blob/master/loss.py#L83) function. 



> * Is the loss function the same as the YoloV3 paper? since I tried it with a real dataset and I'm having a little trouble training the model, so that idea got into my mind.

Actually in the Yolov3 paper, it didn't mention its loss function, right?  So, I use yolov2 loss at each scale, which you can find in this keras [repo](https://github.com/allanzelener/YAD2K/blob/master/yad2k/models/keras_yolo.py#L152). 

BTW, as I commented in the repo, I didn't actually train it in real dataset, because I don't have a decent GPU, sad :). So there might be some bugs in the loss function, you can cross-check with this keras [repo](https://github.com/qqwweee/keras-yolo3/blob/master/yolo3/model.py#L345).

**Comment by ghost** [comment]
Got it, thank you so much for your time!
